### DISCUSSION TITLE
Is airflow FIPS compliant?

### DISCUSSION THREAD

**Discussion by rafiqkattangere** [apache/airflow#38475] [Q&A]
Is apache airflow FIPS compliant? 
This question keeps coming up in discussion with users.  I am aware of changes made in #31171 to run in FIPS compliant environment. But what about airflow and its dependent python packages? Any insights or directions anyone can share please.

**Comment by potiuk** [comment]
I think if you want to make an audit and prove the compliance, Airflow community would welcome such an official compliance check paid by one of the companies that are interested in it. Airflow is a free OSS software, so contributions of any kind like that are welcome. As everything here, the software is contributed by individual volunteers. But companies who want to contribute making an audit and compliance are welcome to contribute it too. 

Would your company be willing to pay for such a compliance check and contribute it back?  Because this is the only way answer can be given to your question - by someone paying for such an audit.

**Reply by rafiqkattangere** [reply]
Thanks for the suggestion. Its above my pay grade to comment on that. 
I was hoping to leverage any technical guidance from community on FIPS compliance -  like are there any code scanners to provide insights into FIPS, how do we deal with nested package dependencies etc.

**Reply by potiuk** [reply]
Sure. That's another angle of it. I'd love to hear about that as well.
